### PR TITLE
docs(react-sdk): Reaction component

### PR DESCRIPTION
**Pull request by arnautov-anton**

### CODE REVIEW

File: packages/react-sdk/src/components/Reaction/Reaction.tsx [hunk 1]
```diff
@@ -15,33 +13,32 @@ export const defaultEmojiReactions: Record<string, string> = {
...
-    <div
-      className={clsx(
-        'str-video__reaction',
-        isShowing && 'str-video__reaction--visible',
-      )}
-    >
+    <div className="str-video__reaction">
       <span className="str-video__reaction__emoji">
-        {emoji_code && defaultEmojiReactions[emoji_code]}
+        {emojiCode && defaultEmojiReactions[emojiCode]}
```
Comment (MartinCupela) [inline packages/react-sdk/src/components/Reaction/Reaction.tsx:41]: Don't we want to make the emoji set customizable? the `default` part of the `defaultEmojiReactions` variable would imply that.

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx [hunk 2]
```diff
@@ -3,8 +3,48 @@ id: reaction
...
+    <>
+      <Reaction participant={participant} hideAfterTimeoutInMs={5000} />
+    </>
+  );
+};
+```
+
+## Final steps
+
+Now we can pass this custom `ParticipantViewUI` component down to our call layout components or directly to `ParticipantView` component in our custom call layout as described in the [participant view customizations guide](../../../ui-cookbook/participant-view-customizations).
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx:32]: Not sure if we want to capitalize the guides' names. For example in the intro section, the guides names are capitalized.

**Review by MartinCupela — APPROVED** [review]

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx [hunk 3]
```diff
@@ -3,8 +3,48 @@ id: reaction
 title: Reaction
 ---
 
-:::warning
+Reaction component is used to display emojis in real-time for a specified amount of time. You can utilise this functionality to notify other participants that you want to speak by "raising hand". Our default sorting algorithm will push participants with raised hand to the top of the list for better visibility. Learn more about reaction events and their customization in the [Reactions & Custom Events guide](../../../guides/reactions-and-custom-events) and see how sorting works in the [Participant sorting guide](../../../guides/sorting-api).
```
Comment (szuperaz) [inline packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx:6]: I'd list the supported reactions here, reading that paragraph made me think that it's for raising hand only.

**Review by szuperaz — APPROVED** [review]

File: packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx (hunk 3 above)
Comment (arnautov-anton) [reply packages/react-sdk/docusaurus/docs/React/03-ui-components/utility/reaction.mdx:6]: Good point, will add, thanks! 👍

**Review by oliverlaz — APPROVED** [review]

**Pull request merged**